### ISSUE TITLE
challenge problem 4.8 sampleoutput missing

### ISSUE THREAD

**Issue by probinso** [GaloisInc/ppaml-cp4#5]
There seems to be a **File Format** section in the 4.8 documentation, but It would be very helpful for the teams if there was sample output provided in the ppaml-cp4/sampleoutput  directory for cp4.8 .

**Comment by nimar** [comment]
Hi Philip,

All the bulletins have the same format, so they can be considered as 
sampleoutput. In fact, there is a solution file which could be 
considered sampleoutput.

:-) Nim

**Comment by slater-ppaml** [comment]
We may wish to add a sample output from the evaluation script.  It's not 
hugely useful, but it does show people what the eval output should look 
like.

Michael

**Comment by nimar** [comment]
Yup, this is already there in the documentation at the end of section 
4.2, which is titled Evaluation. It is also there in appendix C.4 titled 
"evaluate.py".

:-) Nim
